packages/core/src/RPA/core/notebook.py

```diff
@@ -109,6 +109,19 @@ def notebook_table(table, count: int = 20):
         display(Markdown(output))
 
 
+@print_precheck
+def notebook_table_from_raw_data(data, columns=None, index=None, count=None):
```

Comment:
Fair. The autoconversion sounds sensible to me, biggest reason I didn't do that is that it would be a real API change. Also I personally found the Table() constructor's API pretty confusing in how it can take data in 3 different formats on the same argument. But adding proper type signature to it would make it easier to read, I can also do that change in this PR actually.